jinja2/base.html

```diff
@@ -210,6 +210,46 @@
   {% block content %}{% endblock %}
   </div></main>
 
+  <!-- redesign notices -->
```

Comment:
Nit: move this block to an include, like ``jinja2/includes/redesign_notice.html``, and use ``{% include "includes/redesign_notice.html" %}``